### DISCUSSION TITLE
Problem to carry out a DEM on GCD

### DISCUSSION THREAD

**Discussion by ceregepaul** [Riverscapes/gcd#422] [I need help please] [answered]
I'm working in SIGeo in CEREGE ( European Center Research And Teaching Environmental Geosciences) in Michal Tal team.

First of all, it seems to have a problem when we put this DEM in GCD. No data's lines are generated. However, in ArcMap or another sig software, the DEM is continous and values are right. 

![image](https://user-images.githubusercontent.com/95764115/145237347-015961bc-d5ed-417e-a483-aff8aed163e4.png)

Then, when we carry out this DoD, we get unexploitable results (see GCD_DoD).

![image](https://user-images.githubusercontent.com/95764115/145238374-c2a4aa54-3258-4d15-8bda-3b8d53b44c4f.png)

However, when we execute this DoD with ArcMap (using Raster Calculator), we obtain a correct result.

![image](https://user-images.githubusercontent.com/95764115/145238765-c72b887f-7fd4-4701-9de6-d08885db701e.png)

We don't understand where the problem comes from. Could you please help us ?

Please find attached in this message the two DEMs that allowed us to make this DoD.
For your information, please note that the DEMs and the DODs we send you are small samples extracted from the full data file.

Hoping you can help me,

Paul RE
[dem.zip](https://github.com/Riverscapes/gcd/files/7677192/dem.zip)
[GCD_DoD.zip](https://github.com/Riverscapes/gcd/files/7677195/GCD_DoD.zip)
[RasterCalculator_DoD.zip](https://github.com/Riverscapes/gcd/files/7677201/RasterCalculator_DoD.zip)

**Comment by joewheaton** [comment, marked as answer]
Hi @ceregepaul and Michal,
So you have two problems and both are related to a requirement of GCD that all rasters are [orthogonal and dimensionally consistent](http://gcd.riverscapes.xyz/Concepts/data-preparation---best-practices/) (i.e. their width and heights can be evenly divisible by cell resolution). The first problem is that both rasters are NOT the same resolution - one is 15 cm and the other is 20 cm. The second problem is that these are poorly formed rasters with arbitrary extents that are based on the vector data instead of rounded up to something nice and evenly divisible by 0.2. My suggestion is to make both rasters from the raw topographic data again and exert control over the cell resolution and extents and make both exactly the same. 

[![Video Link https://www.youtube.com/watch?v=Vwx7hDvEqjo](http://img.youtube.com/vi/Vwx7hDvEqjo/0.jpg)](http://www.youtube.com/watch?v=Vwx7hDvEqjo "")

Shout if you are unable to figure it out. This is what is explained  [here](http://gcd.riverscapes.xyz/Concepts/data-preparation---best-practices/)

Also, thank you for providing raw data as it made this easy to troubleshoot.
